### ISSUE TITLE
Treemap with null value crashes on legend in Highcharts 5

### ISSUE THREAD

**Issue by kzoon** [highcharts/highcharts#6394]
In a treemap with null values hovering on the corresponding item in the legend crashes in treemap.js.

To reproduce:  http://jsfiddle.net/kzoon/41wz0prs/2/
Hover over the last series in the legend

Code crashes on 
```
setState: function (state) {
		H.Point.prototype.setState.call(this, state);
		this.graphic.attr({
			zIndex: state === 'hover' ? 1 : 0
		});

```
because this.graphic is undefined

**Comment by pawelfus** [comment]
Thanks @kzoon for reporting this issue!

**Comment by jimczj** [comment]
I want to handle this bug, so give me a few day to pull request.Thanks.

**Comment by jimczj** [comment]
When I was fixing bug, I found that when you click the legend,the legend's color will change to Grey,but when you click again,it cannot restore its original color.

**Comment by jon-a-nygaard** [comment]
Not a duplicate, but this issue is managed and fixed together with #6402
